### PR TITLE
Feat/contract uri

### PR DESCRIPTION
**Pull request by Th0rgal**
This adds a contractURI function to the contract. To avoid having duplicate code I added a function to pass storage_variable as function parameter.

### CODE REVIEW

File: tools/deploy.py [hunk 1]
```diff
@@ -19,7 +19,8 @@
 starkpath_public_key = (
     700643851346402816804472395908209287091992718724213663534740619177753103074
 )
-token_uri_base = "https://goerli.starknet.quest/api/quests/starkfighter/uri?level="
+token_uri_base = "https://api.goerli.starknet.quest/quests/uri?level="
+contract_uri = "https://api.goerli.starknet.quest/quests/contract_uri"
```
Comment (fricoben) [inline tools/deploy.py:23]: Are you sure these are the new URI ? I have nothing when I go on these links

Comment (fricoben) [reply tools/deploy.py:23]: this is the URI that I created http://goerli.starknet.quest/api/quests/uri on my end

**Review by fricoben — CHANGES_REQUESTED** [review]
Good just want a verification here

File: tools/deploy.py (hunk 1 above)
Comment (Th0rgal) [reply tools/deploy.py:23]: It will be the new URI when the rust server is deployed.

**Review by fricoben — APPROVED** [review]

**Pull request merged**